src/components/controls/TextInput/README.md

```diff
@@ -268,3 +268,20 @@ LANDING_BLOCK-->
 | validationState | Validation state                                                                                                        |                              `"invalid"`                              |                 |
 | value           | The value of the control                                                                                                |                               `string`                                |                 |
 | view            | The view of the control                                                                                                 |                         `"normal"` `"clear"`                          |   `"normal"`    |
+
+## CSS API
+
+| Name                                       | Description                   |
+| :----------------------------------------- | :---------------------------- |
+| `--g-text-input-border-radius`             | Border radius                 |
+| `--g-text-input-text-color`                | Text color                    |
+| `--g-text-input-placeholder-color`         | Placeholder color             |
+| `--g-text-input-disabled-background-color` | Disabled background color     |
+| `--g-text-input-border-color`              | Border color                  |
+| `--g-text-input-border-color-hover`        | Border color hovered          |
+| `--g-text-input-border-color-active`       | Border color active           |
+| `--g-text-input-line-height`               | Line-height                   |
+| `--g-input-height`                         | Height                        |
```

Comment:
I think we do not need this variable, because of our design system. We provide `size` property to control height

Reply:
This is also true for line-height variable, we don't need it

Reply:
Button component has a similar property `--g-button-height`. Maybe we should add this property for consistency? If `--g-input-height` is defined, the component will ignore size prop

Reply:
The cases of Button and TextInput aren't the same. How we handle label and clear button when this var is set? What size clear button should be?